packages/next-yak/loaders/lib/__tests__/extractCssUnit.test.ts

```diff
@@ -0,0 +1,39 @@
+import extractCSSUnit from "../extractCssUnit.cjs";
+import { describe, it, expect } from "vitest";
+
+describe("extractCssUnit", () => {
+  it("should handle all possible cases", () => {
+    const testCases = [
+      " ", // => null
+      ";\n  display: block;\n  ", // => null
+      " ease-out 1s infinite reverse both paused slidein;\n", // => ease-out
+      ";\n    ", // => null
+      "rem;\n        margin: ", // => rem
+      "px;\n        z-index: ", // => px
+      ";\n        transform: translateX(", // => null
+      "px) translateY(", // => px
+      "px);\n        font-family: 'Arial', sans-serif;\n     ", // => px
+      "deg)\n          translate(0, -88px) rotate(", // => deg
+      "deg);\n     ", // => deg
+    ];
+
+    const result = testCases.map((cssString) => {
+      return extractCSSUnit(cssString);
```

Comment:
maybe this could improve the readability of the result as we always see input -> output

```suggestion
      return { code: cssString, unit: extractCSSUnit(cssString) };
```